Remotion/ObjectBinding/Web/Res/Themes/NovaGray/Html/BocList.css

```
@@ -400,6 +400,22 @@ tr.bocListDataRow td.bocListDataCell span.CommandText
 
 
 /* Style to override the BocList's anchor style for drop down menus rendered inside a cell. */
+td.bocListDataCell div ul.DropDownMenuOptions li a
+{
+  display: block;
+}
+
+td.bocListDataCell div ul.DropDownMenuOptions li.DropDownMenuItemDisabled.selected a,
+td.bocListDataCell div ul.DropDownMenuOptions li.DropDownMenuItemDisabled a:active,
+td.bocListDataCell div ul.DropDownMenuOptions li.DropDownMenuItemDisabled a:focus,
+td.bocListDataCell div ul.DropDownMenuOptions li.DropDownMenuItemDisabled a:hover
+{
+  background-color: #ECEEEC;
+  border-style: none;
+  padding: 2px 2px 2px 2px;
```

I may be missing somehting (only looked at the DropDownMenu.css), but line 287ff 
`ul.DropDownMenuOptions li.DropDownMenuItemDisabled a`
says that padding uses `1px 3px 3px 3px`, here it's 2px?

---

Imo a padding of `2px` is more correct anyway, as non-disabled anchors have a padding of `1px` + the anchor with a width of `1px`, so the disabled one needs a padding of `2px`. The question is rather why the other file sets `1px 3px 3px 3px`.

Anyway, the line should be changed to 
```suggestion
  padding: 2px;
```

---

Very true. But given that I don't know rihgt now, what the bug is, plase use the original value from DropDownMenu.css to keep things consistent, even if it's strange/wrong. Otherwise, we will eventually confronted with the question, why there's a difference and what's the correct version.

---

That would need other CSS rule additions in this file. Using the proposed values leads to style induced movement of the anchor element when hovering.

---

ah, okay. you win :)